test/browser_tests/default-suites.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var settings = require( './settings.js' );
+
+var defaultSuites = {
+  // Set default browser suites to test.
+  // These values are passed to the `multiCapabilities` property
+  // of the protractor config object.
+  // See https://docs.saucelabs.com/reference/platforms-configurator/
+  // for configuration values for version and platform.
+
+  // Essential browsers for running locally.
+  essential: [
+    {
+      specs:       [ settings.specsBasePath + '*.js' ],
+      browserName: 'chrome',
+      version:     '',
+      platform:    'WINDOWS',
+      maxDuration: 10800
+    }
+  ],
+
+  // Legacy browsers to run in the cloud.
+  legacy: [
+    {
+      specs:       [ settings.specsBasePath + '*.js' ],
+      browserName: 'internet explorer',
+      version:     '8.0',
+      platform:    'XP',
+      maxDuration: 10800
+    }
+  ],
+
+  // Modern browsers to run in the cloud.
+  modern: [
+    {
+      specs:       [ settings.specsBasePath + '*.js' ],
+      browserName: 'firefox',
+      version:     '',
+      platform:    'WINDOWS 10',
+      maxDuration: 10800
+    },
+    {
+      specs:       [ settings.specsBasePath + '*.js' ],
+      browserName: 'internet explorer',
+      version:     '',
+      platform:    'WINDOWS 10',
+      maxDuration: 10800
+    }
+  ]
+};
+
+// Run all browsers together.
+defaultSuites.full = defaultSuites.essential.concat(
```

Comment:
Mentioned in chat, but for completeness https://github.com/cfpb/cfgov-refresh/pull/859/files#diff-bde509149b84f69921b6afc2471eceafR89 ... also adjusted in https://github.com/cfpb/cfgov-refresh/commit/7825f9ad0a48eba12762cf4343013926c0c6f9b2